lib/permutations.rb

```diff
@@ -1,4 +1,20 @@
 
 def permutations?(string1, string2)
-  raise NotImplementedError, "permutations? not implemented"
+  hash = {}
+  string1.each do |letter|
```

Comment:
Strings don't have a .each, either split them or do 

```suggestion
  string1.each_char do |letter|
```